src/sql/azdata.proposed.d.ts

```diff
@@ -1402,16 +1402,38 @@ declare module 'azdata' {
 		stepDetails: AgentJobStepInfo;
 	}
 
+	export enum AgentSubSystem {
```

Comment:
Suggest adding a unit test for these as well that ensures this and the sqlExtHostTypes version are kept in sync (also add a comment here explaining that)